stash/test/api/collators.api.test.ts

```diff
@@ -41,16 +41,15 @@ function validateReturnedDate(date :string, format : string) {
 
 //perhaps those addresses may change when data is upgraded!
 
-const oldCollatorAddress = "5EtT1Psa48f9KSNMZqZgwPj6guNULVeRUzcA9w1Kk8DkP9iZ";
-const collatorAddress =  "5HNnu1WVhPCxdD8uHFoRyZ2qJE8hWQL3aaG8D1gaYhK3j8a7";
+const collatorAddress =  "0xf24ff3a9cf04c71dbc94d0b566f7a27b94566cac";
 
 function validateValidApyRange(body) {
     const apy = parseFloat(body.apy)
     expect(apy).to.be.lessThanOrEqual(40)
     expect(apy).to.be.greaterThan(10)
 }
 
-describe.skip('APi tests: Collator apy - dailyRewards', () => {
+describe('APi tests: Collator apy - dailyRewards', () => { //todo: remove skip once we know new collator addresses
```

Comment:
Tests skipped until we know the right collator address and that address has data.

Reply:
We have the right address, but no data still - will remove skip when we have the data.